lessons/beginners/install-editor/notepad-plus-plus.md

```diff
@@ -17,7 +17,7 @@ a nainstaluj.
 {% block setup %}
 
 Odsazování
-:   V menu Nastavení zvol Předvolby a pak nastav
+:   V menu Nastavení zvol Volby a pak Syntaxe a pak nastav
```

Comment:
Just a tiny detail, the part _a pak_ is there twice in the same sentence, I would change it to something different. However, if you have a better idea of what should be there or you are OK with having _a pak_ twice in one sentence, I am OK with that and I approve the change.
```suggestion
:   V menu Nastavení zvol Volby potom Syntaxe a pak nastav
```